Use installed Abseil 20250512.x with ONNX Runtime 1.26.0.

ONNX Runtime 1.26.0 expects Abseil 20250814.0 and uses an ABSEIL_LIBS
list generated for that release. The foss/2025a stack provides Abseil
20250512.1. Accept the 20250512 LTS family and use the ABSEIL_LIBS list
from ONNX Runtime 1.23.2, which targets that Abseil release.

Author: Pavel Tomanek (Inuits/UGent) with help from ChatGPT5.6
--- a/cmake/external/abseil-cpp.cmake
+++ b/cmake/external/abseil-cpp.cmake
@@ -37,7 +37,7 @@
     URL_HASH SHA1=${DEP_SHA1_abseil_cpp}
     EXCLUDE_FROM_ALL
     PATCH_COMMAND ${ABSL_PATCH_COMMAND}
-    FIND_PACKAGE_ARGS 20250814 NAMES absl
+    FIND_PACKAGE_ARGS 20250512 NAMES absl
 )
 
 onnxruntime_fetchcontent_makeavailable(abseil_cpp)
@@ -62,87 +62,98 @@
 
 # TODO: since multiple ORT's dependencies depend on Abseil, the list below would vary from version to version.
 # We'd better to not manually manage the list.
-# This list is generated using tools/python/resolve_absl_deps_dynamic.py for Abseil version 20250814.0
-# The libraries are topologically sorted for correct static linking order.
 set(ABSEIL_LIBS
+absl::absl_log
+absl::log_internal_log_impl
+absl::log_internal_strip
+absl::log_internal_message
+absl::log_internal_format
 absl::synchronization
-absl::tracing_internal
+absl::str_format
+absl::flags
+absl::log_internal_globals
+absl::kernel_timeout_internal
+absl::str_format_internal
+absl::hash
+absl::log_internal_append_truncated
+absl::absl_vlog_is_on
+absl::flags_commandlineflag
 absl::time
+absl::symbolize
+absl::graphcycles_internal
+absl::log_internal_conditions
+absl::strings
+absl::malloc_internal
+absl::demangle_internal
+absl::optional
+absl::stacktrace
+absl::base
+absl::demangle_rust
+absl::bad_optional_access
+absl::strings_internal
+absl::debugging_internal
+absl::int128
+absl::spinlock_wait
+absl::decode_rust_punycode
+absl::raw_logging_internal
+absl::flat_hash_set
+absl::flat_hash_map
+absl::node_hash_map
+absl::node_hash_set
+absl::compare
+absl::base_internal
+absl::nullability
+absl::bounded_utf8_length_sequence
+absl::log_severity
+absl::type_traits
+absl::atomic_hook
+absl::bits
+absl::flags_commandlineflag_internal
+absl::hash_container_defaults
+absl::numeric_representation
+absl::node_slot_policy
+absl::core_headers
+absl::dynamic_annotations
+absl::utf8_for_code_point
+absl::errno_saver
+absl::absl_check
+absl::hash_function_defaults
+absl::function_ref
+absl::city
+absl::low_level_hash
+absl::fixed_array
+absl::variant
+absl::meta
+absl::log_internal_voidify
+absl::log_sink
+absl::log_internal_log_sink_set
+absl::log_sink_registry
+absl::log_entry
+absl::log_globals
+absl::log_internal_nullguard
+absl::examine_stack
+absl::inlined_vector
+absl::log_internal_proto
+absl::strerror
+absl::log_internal_config
+absl::raw_hash_map
+absl::raw_hash_set
+absl::container_memory
+absl::algorithm_container
+absl::span
+absl::log_internal_nullstream
+absl::vlog_config_internal
+absl::flags_reflection
+absl::flags_internal
+absl::flags_config
+absl::fast_type_id
+absl::utility
 absl::time_zone
 absl::civil_time
-absl::symbolize
-absl::demangle_internal
-absl::demangle_rust
-absl::stacktrace
-absl::debugging_internal
-absl::malloc_internal
-absl::kernel_timeout_internal
-absl::graphcycles_internal
-absl::str_format
-absl::str_format_internal
-absl::flat_hash_set
-absl::flat_hash_map
-absl::algorithm_container
-absl::raw_hash_map
-absl::raw_hash_set
-absl::prefetch
-absl::hashtablez_sampler
-absl::hashtable_debug_hooks
-absl::hashtable_control_bytes
-absl::hash_policy_traits
-absl::common_policy_traits
-absl::hash_container_defaults
-absl::hash_function_defaults
-absl::cord
-absl::inlined_vector
-absl::inlined_vector_internal
-absl::span
-absl::crc_cord_state
-absl::crc32c
-absl::cordz_update_tracker
-absl::cordz_update_scope
-absl::cordz_info
-absl::cordz_functions
-absl::cord_internal
-absl::container_common
-absl::container_memory
-absl::hash
-absl::variant
-absl::optional
-absl::strings
+absl::string_view
+absl::throw_delegate
+absl::memory
 absl::charset
-absl::strings_internal
-absl::string_view
-absl::int128
-absl::compare
-absl::function_ref
-absl::any_invocable
-absl::city
-absl::bits
 absl::endian
-absl::flags
-absl::fixed_array
-absl::weakly_mixed_integer
-absl::memory
-absl::meta
-absl::throw_delegate
-absl::iterator_traits_internal
-absl::algorithm
-absl::compressed_tuple
-absl::utility
-absl::base
-absl::type_traits
-absl::spinlock_wait
-absl::raw_logging_internal
-absl::errno_saver
-absl::nullability
-absl::log_severity
-absl::dynamic_annotations
-absl::base_internal
-absl::atomic_hook
-absl::core_headers
-absl::config
-absl::absl_log
-absl::log_internal_log_impl
-absl::absl_check
-absl::log_internal_check_impl)
+absl::config)
+
